sources/Inputs/src/Node.cpp

```diff
@@ -67,5 +68,61 @@ operator>=(const Node& lhs, const Node& rhs) {
 
 VoltageLevel::VoltageLevel(const VoltageLevelId& vlid) : id(vlid) {}
 
+/////////////////////////////////////////////////
+
+std::shared_ptr<Line>
+Line::build(const LineId& lineId, const std::shared_ptr<Node>& node1, const std::shared_ptr<Node>& node2) {
```

Comment:
It is very misleading to have node1 and node2 const whereas we edit the referenced value...

Reply:
OK